higher_health/views.py

```diff
@@ -42,6 +43,17 @@ def get_context_data(self, **kwargs):
         )
         return ctx
 
+    def dispatch(self, request, *args, **kwargs):
+        if self.request.user.is_authenticated and self.request.method == "GET":
+            triage = Covid19Triage.objects.filter(user=self.request.user).last()
+            if (
+                triage
+                and triage.timestamp.date() >= datetime.now().date()
+                and "redo" not in self.request.GET
```

Comment:
I don't see anywhere in the templates where we have a `redo` link. Is this something that's just for testing?

Reply:
No, cross PR - it's added in #91

Reply:
@rudigiesler Let me add a test for it though